src/modules/file-preview-card.marko

```diff
@@ -0,0 +1,535 @@
+                        <video src="https://storage.googleapis.com/gtv-videos-bucket/sample/ForBiggerMeltdowns.mp4"/>
+                    </div>
+                    <div class="file-preview-card__menu-action">
+                        <button
+                            class="icon-btn"
+                            type="button"
+                            aria-label="Open menu"
+                        >
+                            <svg
+                                class="icon icon--16"
+                                height="16"
+                                width="16"
+                                aria-hidden="true"
+                            >
+                                <use href="#icon-overflow-vertical-16"/>
+                            </svg>
+                        </button>
+                    </div>
+                    <div class="file-preview-card__preview-action">
+                        <button class="icon-btn" type="button">
+                            <svg
+                                class="icon icon--16 file-preview-card__video-icon"
+                                aria-hidden="true"
+                            >
+                                <use href="#icon-play-16"/>
+                            </svg>
+                            <span class="file-preview-card__preview-action-text">
+                                0:21
+                            </span>
+                        </button>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <highlight-code type="html">
+        <div class="file-preview-card">
+            <div class="file-preview-card__body">
+                <img
+                    src="https://ir.ebaystatic.com/cr/v/c01/skin/docs/tb-real-square-pic.jpg"
+                    alt="Image"
+                >
+            </div>
+            <div class="file-preview-card__menu-action">
+                <button class="icon-btn" type="button" aria-label="Open menu">
+                    <svg
+                        class="icon icon--16"
+                        height="16"
+                        width="16"
+                        aria-hidden="true"
+                    >
+                        <use href="#icon-overflow-vertical-16"/>
+                    </svg>
+                </button>
+            </div>
+            <div class="file-preview-card__preview-action">
+                <button class="icon-btn" type="button">
+                    <svg
+                        class="icon icon--16 file-preview-card__video-icon"
+                        aria-hidden="true"
+                    >
+                        <use href="#icon-play-16"/>
+                    </svg>
+                    <span class="file-preview-card__preview-action-text">
+                        0:21
+                    </span>
+                </button>
+            </div>
+        </div>
+    </highlight-code>
+
+    <h3>Non image</h3>
```

Comment:
I preferred "Document" here, rather than "Non-image". Not sure if there was a reason.

Reply:
I wanted to be generic in case we have other type of file types that are not image specific and also not documents, can change it to Document